book/src/developer-guides/node-setup-guide/build-from-source.md

```diff
@@ -0,0 +1,59 @@
+## Build from source
+
+In this step we will setup rust compiler, toolchain and build a node. 
+
+**Setup required libraries**
+
+```bash
+sudo apt install -y git clang curl libssl-dev llvm libudev-dev
+```
+**Setup Rust binary and Toolchain**
+
+```bash
+#!/bin/bash
+
+RUST_C="nightly-2021-11-07"
```

Comment:
we are using [nightly-2022-04-18](https://github.com/ComposableFi/composable/blob/main/docker/ci-linux.dockerfile#L5) now